README.md

```diff
@@ -515,15 +522,16 @@ Using the private key:
     const signedMessage = await openpgp.readCleartextMessage({
         cleartextMessage // parse armored message
     });
-    const verified = await openpgp.verify({
+    const verificationResult = await openpgp.verify({
         message: signedMessage,
         verificationKeys: publicKey
     });
-    const { valid } = verified.signatures[0];
-    if (valid) {
-        console.log('signed by key id ' + verified.signatures[0].keyID.toHex());
-    } else {
-        throw new Error('signature could not be verified');
+    const { verified, keyID } = verificationResult.signatures[0];
+    try {
+        await verified; // throws on invalid signature
+        console.log('signed by key id ' + keyID.toHex());
```

Comment:
The [also below](https://github.com/openpgpjs/openpgpjs/pull/1348#discussion_r658649826) was significant :upside_down_face:
```suggestion
        console.log('Signed by key id ' + keyID.toHex());
```